src/pybind/mgr/cephadm/module.py

```diff
@@ -2252,9 +2254,19 @@ def matches_network(host):
             self.log.debug('Placing %s.%s on host %s' % (
                 daemon_type, daemon_id, host))
 
-            daemon_spec = self.cephadm_services[daemon_type].prepare_create(daemon_spec)
-
-            self._create_daemon(daemon_spec)
+            try:
+                daemon_spec = self.cephadm_services[daemon_type].prepare_create(daemon_spec)
+                self._create_daemon(daemon_spec)
+                r = True
```

Comment:
```suggestion
                r |= True
```

nit, but would using a bitwise operator be simpler?

Reply:
I don't think so, unless I misunderstand; the idea is that any of the various add or remove operations can cause a definitive True or False, but "no operation" should also return False, so that should only happen if none of the loops did anything at all.